dags/worker_op.py

```diff
@@ -17,4 +17,5 @@ def worker_op(**kwargs):
         execution_timeout=kwargs.get("execution_timeout", None),
         queue=kwargs["queue"],
         dag=kwargs["dag"],
+        qos=kwargs.get("qos", True),
```

Comment:
What does "qos" do?

Reply:
This is a switch turning the container timeout after 5 minutes on/off. 
https://github.com/seung-lab/seuron/blob/main/plugins/custom/docker_custom.py#L188-L190